rust/src/transformations/make_stable_expr/expr_binary/test.rs

```diff
@@ -135,6 +135,18 @@ fn test_eq() -> Fallible<()> {
     )
 }
 
+#[test]
+fn test_eq_missing() -> Fallible<()> {
+    test_binary!(
+        get_f64_data,
+        eq_missing,
+        [
+            true, false, // NaN is equal to NaN?
```

Comment:
This is surprising, and doesn't seem to be the case in other implementations. Python for example:
```
>>> float('nan') == float('nan')
False
```
I think this is also the IEEE standard.

I'd suggest filing an issue and linking to it here (and in the test above), and then understanding where this behavior comes from. Even if it's not a mistake, there should still be some explanation of this in the polars docs?

Reply:
It's a documented departure from IEEE754. I think they'd rather run a (fast) bitwise comparison.